packages/blockchain-wallet-v4-frontend/src/scenes/Exchange/Shapeshift/selectors.js

```diff
@@ -0,0 +1,19 @@
+import { selectors } from 'data'
+import { lift } from 'ramda'
+
+export const getData = state => {
+  const userCountryR = selectors.core.settings.getCountryCode(state)
+  const userStateR = selectors.core.kvStore.shapeShift.getUsState(state)
+  const exchangeStep = selectors.components.exchange.getStep(state)
+
+  const transform = (country, usState) => {
+    // determine if user is in the US and has not registered their state with Shapeshift
+    const isRegistered = country === 'US' && usState && usState.Code
```

Comment:
Only user's in the US have to select a state. If a user from outside US who has been using shapeshift accesses Exchange, or someone outside US accesses the page for the first time isRegistered will be false and step will be 0. I think the check for isRegistered should be `country !== 'US' || usState`

Reply:
fixed